components/org.wso2.carbon.identity.branding.preference.management.core/src/main/java/org/wso2/carbon/identity/branding/preference/management/core/internal/BrandingPreferenceManagerComponentDataHolder.java

```diff
@@ -48,4 +50,24 @@ public void setConfigurationManager(ConfigurationManager configurationManager) {
 
         this.configurationManager = configurationManager;
     }
+
+    /**
+     * Return identity event service.
```

Comment:
Get identity event service reference.

Reply:
Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/pull/3/commits/6c642e68405f6962618070bef2176e84b66cf5c0

Reply:
I don't see this updated yet.

Reply:
Adressed with https://github.com/wso2-extensions/identity-branding-preference-management/pull/3/commits/35cccfb9edb1ac51496089e3de3c3c7ba2745443